angular/index.d.ts

```diff
@@ -97,7 +97,7 @@ declare namespace angular {
          * @param iterator Iterator function.
          * @param context Object to become context (this) for the iterator function.
          */
-        forEach<T>(obj: T[], iterator: (value: T, key: number) => any, context?: any): any;
+        forEach<T>(obj: T[], iterator: (value: T, key: number, obj?: T[]) => any, context?: any): any;
```

Comment:
I have also changed return type of `forEach`. It should return reference to `obj`.